### PR TITLE
New about page string keys added

### PR DESCRIPTION
**Pull request by renderf0x**
Adds strings for the new page introduced in [the frontend repo](https://github.com/ourjapanlife/findadoc-frontend/pull/251), and deletes old about page string content.

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -1,16 +1,87 @@
...
-        "trustQ": "Can this information be trusted?",
-        "welcome": "Welcome to Find a Doc, Japan!",
-        "whatIsThisA": "Find a Doc is a community-driven database that was created by LaShawn Toyoda to help spread information regarding healthcare services in Japan. The project has since grown into a team of volunteers, and currently we're focusing on keeping the immigrant community informed about clinics that are offering waiting lists for vaccinations so they don't go to waste.<br /><br />In the future, we'd like to expand Find a Doc to help non-Japanese speaking residents of Japan find clinics that offer services in their native language.<br /><br /> If you find this project helpful in any way, <a href='https://ko-fi.com/theyokohamalife'><b>please donate</b></a> to help us keep the site running and to continue adding new features.",
-        "whatIsThisQ": "What is this?"
+        "header": {
+            "headline": "About Find A Doc, the database reducing vaccine waste."
```
Comment (theyokohamalife) [inline locales/en.json:4]: `About Find a Doc, the health database for Japan's international community.`

File: locales/en.json [hunk 2]
```diff
@@ -1,16 +1,87 @@
...
+        },
+        "kofi": {
+            "headline": "Buy us coffee",
+            "text": {
+                "0": "We are running this database after our 9-5 jobs. Help us stay awake!",
+                "1": "Consider supporting us if you find this project helped you or someone in your network."
```
Comment (theyokohamalife) [inline locales/en.json:10]: `Consider supporting us if this project has helped you or someone you know.`

File: locales/en.json [hunk 3]
```diff
@@ -1,16 +1,87 @@
...
+            },
+            "action": "Donate a cuppa"
```
Comment (theyokohamalife) [inline locales/en.json:12]: `Donate`

File: locales/en.json [hunk 4]
```diff
@@ -1,16 +1,87 @@
...
+                "action": "I want to join!"
+            }
+        },
+        "timeline": {
+            "items": {
+                "0": {
+                    "date": "2021.06.17",
+                    "dateHeadline": "From idea to health database",
+                    "text": {
+                        "0": "On that day, LaShawn Toyoda had enough of the lack of direction from officials. A few hours later, Find a Doc is born.",
```
Comment (theyokohamalife) [inline locales/en.json:38]: `LaShawn Toyoda had enough about the lack of clear information regarding the vaccine rollout in Japan from governing officials. A few hours later, Find a Doc was born.`

File: locales/en.json [hunk 5]
```diff
@@ -1,16 +1,87 @@
...
+            "join": {
+                "headline": "Does Find a Doc’s mission spark your interests?",
+                "subheadline": "Join us over on GitHub to see how you can contribute!",
+                "action": "I want to join!"
+            }
+        },
+        "timeline": {
+            "items": {
+                "0": {
+                    "date": "2021.06.17",
```
Comment (theyokohamalife) [inline locales/en.json:35]: `2021.06.13`

File: locales/en.json [hunk 6]
```diff
@@ -1,16 +1,87 @@
...
+                    "dateHeadline": "From idea to health database",
+                    "text": {
+                        "0": "On that day, LaShawn Toyoda had enough of the lack of direction from officials. A few hours later, Find a Doc is born.",
+                        "1": "This database aims to better circulate the latest information on healthcare services in Japan."
```
Comment (theyokohamalife) [inline locales/en.json:39]: This database aims to help keep the international community of Japan informed about critical healthcare services in Japan.

File: locales/en.json [hunk 7]
```diff
@@ -1,16 +1,87 @@
...
+                    },
+                    "textHeadline": "Watch our daughter, I gotta make something."
+                },
+                "1": {
+                    "date": "Presently",
+                    "dateHeadline": "A team of volunteers",
+                    "text": {
+                        "0": "Find a Doc is community-driven with the support of a team of volunteers. Currently, the team is focused on keeping the immigrant community informed about clinics offering waiting lists for vaccinations to reduce waste.",
```
Comment (theyokohamalife) [inline locales/en.json:47]: `Find a Doc is community-driven with the support of a team of volunteers. Currently, the team is focused on keeping the international community of Japan informed about clinics offering waiting lists for vaccinations to reduce waste.`

File: locales/en.json [hunk 8]
```diff
@@ -1,16 +1,87 @@
...
+                        "1":  "In the future, we want Find a Doc to help non-Japanese speaking residents of Japan find clinics that offer services in their native language."
+                    },
+                    "textHeadline": "You can fill the void by citizen action.",
+                    "action": "Check out &/or join the team"
```
Comment (theyokohamalife) [inline locales/en.json:51]: `Join our team`

File: locales/en.json [hunk 9]
```diff
@@ -1,16 +1,87 @@
...
+                }
+            }
+        },
+        "trust": {
+            "cards": {
+                "0": {
+                    "text": {
+                        "0": "All links lead to official resources such as clinic and government websites.",
```
Comment (theyokohamalife) [inline locales/en.json:59]: `We try to make sure that all links lead to official resources such as clinic and government websites.`

File: locales/en.json [hunk 10]
```diff
@@ -1,16 +1,87 @@
...
+                        "1": "Read them thoroughly before registering your personal information with them."
+                    }
+                },
+                "1": {
+                    "text": {
+                        "0": "Anyone in the community can submit content to the database.",
+                        "1": "As a result, is not possible for us to guarantee that accuracy. If you notice any errors, please report the issue to moderators."
```
Comment (theyokohamalife) [inline locales/en.json:66]: `As a result, is not possible for us to guarantee complete accuracy. If you notice any errors, please report them to moderators.`

File: locales/en.json [hunk 11]
```diff
@@ -1,16 +1,87 @@
...
+                    }
+                },
+                "2": {
+                    "text": {
+                        "0": "Our moderators try their best to detect fraudulent information.",
+                        "1": "Even so, research any service that you find on here before visiting the location or submitting your personal information to a third-party website."
```
Comment (theyokohamalife) [inline locales/en.json:72]: `However, we strongly encourage you to do additional research for any services that you find through Find a Doc. Use your best judgment before visiting any of the locations that are listed in the database, or before submitting your personal information to their websites.`

File: locales/en.json [hunk 12]
```diff
@@ -1,16 +1,87 @@
...
+                    }
+                }
+            },
+            "footer": {
+                "headline": "This is an open-source project, and we are not liable for any damages that may be incurred from the use of Find a Doc.",
+                "license": "Read more about it in our license."
+            },
+            "header": {
+                "headline": "Can this information be trusted?",
+                "subheadline": "In short, yes, but do your due diligence"
```
Comment (theyokohamalife) [inline locales/en.json:82]: `Yes, to some degree, but do your due diligence.`

**Review by theyokohamalife — COMMENTED** [review]
I left comments for some strings I'd like changed before the new About page gets translated.

File: locales/en.json (hunk 3 above)
Comment (renderf0x) [reply locales/en.json:12]: Changed but aww I thought that one was cute

File: locales/en.json (hunk 11 above)
Comment (renderf0x) [reply locales/en.json:72]: Changing, but this one is quite long, and may throw off the design as well as be a bit cumbersome to read through. Is there a shorter version?

File: locales/en.json (hunk 12 above)
Comment (renderf0x) [reply locales/en.json:82]: Changed, but I wonder if 'to some degree' might rob the site of credibility?

**Review by theyokohamalife — APPROVED** [review]
LGTM! I left a few comments based on your suggestions for additional changes.

File: locales/en.json (hunk 11 above)
Comment (theyokohamalife) [reply locales/en.json]: `However, we strongly encourage you to use your best judgment and do additional research for any services you discover through Find a Doc.`

File: locales/en.json (hunk 3 above)
Comment (theyokohamalife) [reply locales/en.json]: It was cute! I was just worried about localization. How about `Donate a cup`?

File: locales/en.json [hunk 13]
```diff
@@ -1,16 +1,87 @@
...
+                    }
+                }
+            },
+            "footer": {
+                "headline": "This is an open-source project, and we are not liable for any damages that may be incurred from the use of Find a Doc.",
+                "license": "Read more about it in our license."
+            },
+            "header": {
+                "headline": "Can this information be trusted?",
+                "subheadline": "Yes, to some degree, but do your due diligence."
```
Comment (theyokohamalife) [inline locales/en.json:82]: Hmm, @renderf0x I think you're right. Let's change this line back to what it was originally. `In short, yes, but due your due diligence."

Comment (renderf0x) [reply locales/en.json:82]: ok roger that.

File: locales/en.json (hunk 11 above)
Comment (renderf0x) [reply locales/en.json]: Done

File: locales/en.json (hunk 3 above)
Comment (renderf0x) [reply locales/en.json]: Ok! Changing it to that. I will say that I liked the English teamtime feel of the original, but that's maybe a taste thing

**Pull request merged**